src/app/main/component/user/components/habit/add-new-habit/habit-edit-shopping-list/habit-edit-shopping-list.component.html

```diff
@@ -18,3 +18,9 @@
   <input #newItem id="add-field" type="text" formControlName="item" autocomplete="off" />
   <input #btn [disabled]="!itemForm.valid" id="add-btn" type="submit" value="+" (click)="add(newItem.value); newItem.value = ''" />
```

Comment:
It's not good to have logic inside a template. Move `newItem.value = ''` to `add` method.